### PR TITLE
libuvc: add version 0.0.7

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **libuvc/***

- add version 0.0.7
- add package_type

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libuvc/all/patches/0.0.7-0001-fix-cci-packagename.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/a/CMakeLists.txt b/b/CMakeLists.txt
+index fbaffc9..e14daf4 100644
+--- a/a/CMakeLists.txt
++++ b/b/CMakeLists.txt
+@@ -116,7 +116,7 @@ foreach(target_name IN LISTS UVC_TARGETS)
+   )
+   target_link_libraries(${target_name}
+     # libusb-1.0 used internally so we link to it privately.
+-    PRIVATE LibUSB::LibUSB ${threads}
++    PRIVATE libusb::libusb ${threads}
```
Comment (prince-chrismc) [inline recipes/libuvc/all/patches/0.0.7-0001-fix-cci-packagename.patch:10]: We could remove the patches and have CMakeDeps set the correct name in the generate method too 🙏

**Review by prince-chrismc — COMMENTED** [review]
There's a TODO for PKG_CONFIG that was fixed in 1.55 

https://github.com/conan-io/conan/commit/559bca1bc885f38b5ba8320d0c8d636d32491558

**Comment by prince-chrismc** [comment]
https://github.com/conan-io/conan-center-index/pull/17043 for v2 🤞

**Comment by toge** [comment]
@prince-chrismc 
Sorry, I don't know what to do for this PR.
libusb package doesn't provide LibUSB::LibUSB yet.
Please tell me the way to improve this PR.

**Review by RubenRBS — APPROVED** [review]

**Comment by RubenRBS** [comment]
I had to regenerate libusb packages for this to pass, sorry about that :)

**Review by danimtb — DISMISSED** [review]

File: recipes/libuvc/all/conanfile.py [hunk 2]
```diff
@@ -87,6 +88,9 @@ def generate(self):
         tc = CMakeToolchain(self)
         tc.variables["CMAKE_BUILD_TARGET"] = "Shared" if self.options.shared else "Static"
         tc.variables["LIBUVC_WITH_JPEG"] = bool(self.options.with_jpeg)
+        if Version(self.version) >= "0.0.7":
+            tc.variables["BUILD_EXAMPLE "] = False
```
Comment (RubenRBS) [inline recipes/libuvc/all/conanfile.py:92]: Is the extra space here a typo?

**Review by RubenRBS — CHANGES_REQUESTED** [review]

**Review by RubenRBS — APPROVED** [review]

**Review by danimtb — APPROVED** [review]

**Pull request merged**